Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -117,8 +124,17 @@ - (void)viewDidLoad
     self.filteredEventsArray = [NSMutableArray arrayWithCapacity:self.flatEventsData.count];
 }
 
+-(void)viewWillAppear:(BOOL)animated {
+    
+    [self filterContentForSearchText:@"" scope:
```

Comment:
These lines seem very similar. Is there a way you could create a function? 

`Don't Repeat Yourself`